gpii/configs/preferencesServer.production.json

```diff
@@ -12,7 +12,7 @@
             }
         }
     },
-    "includes": [
+    "mergeConfigs": [
         "../node_modules/preferencesServer/configs/development.json",
         "../node_modules/rawPreferencesServer/configs/production.json"
```

Comment:
should be "../node_modules/rawPreferencesServer/configs/gpii.rawPreferencesServer.config.development.json"